lib/styles/themes/base/def/icons.js

```diff
@@ -1,3 +1,15 @@
 import close from '../icons/close-black.svg';
+import closewhite from '../icons/close.svg';
```

Comment:
can we not use the black icon only? We should be able to change SVG color with fill property

Reply:
Right now we can not use SVG color with fill property because the Icon Component code is SVG Dependent. 
I have some concerns regarding the same. Me and Saurav did some R&D on that. 
As per current code, we can use filter css property for color change.